Header header

geometry_msgs/Quaternion orientation

float32[3] angular_velocity         # [gx, gy, gz] in DPS (degrees per second)

float32[3] linear_acceleration      # [ax, ay, az] in g's (gravitational acceleration)

